samples/caf_sensor_manager/src/modules/workload_sim.c

```diff
@@ -68,15 +68,14 @@ static bool init(void)
 	for (size_t i = 0; i < NUMBER_OF_WORKLOADS; i++) {
 		k_work_init(&workloads[i].work, simulated_work_handler);
 	}
-
-	return false;
 }
 
 static bool event_handler(const struct app_event_header *aeh)
 {
 	if (is_sensor_data_aggregator_event(aeh)) {
 		const struct sensor_data_aggregator_event *event =
 			cast_sensor_data_aggregator_event(aeh);
+			LOG_INF("agg event, data cnt: %d", event->sample_cnt);
```

Comment:
Afair we never logged anything in the handler itself. If you insist to have it, please change to `LOG_DBG`.